Jellyfin.Data.Migrations/JellyfinDbFactory.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Jellyfin.Data.DbContexts;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+
+namespace Jellyfin.Data.Migrations
+{
+    class JellyfinDbFactory : IDesignTimeDbContextFactory<JellyfinDb>
+    {
+        public JellyfinDb CreateDbContext(string[] args)
+        {
+            var optionsBuilder = new DbContextOptionsBuilder<JellyfinDb>();
+            optionsBuilder.UseSqlite("./jellyfin.db");
+
+            return null;
```

Comment:
Looks like this isn't fully implemented yet?

Reply:
Correct, I didn't think it was included in this PR, but the migration code is as of now unused. This code specifically would be used to create instances of `JellyfinDb` at design time, to be used in creating new migrations.